csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,667 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Uptime',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableSentPackets',
+          },
+          properties: [
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Sent packets [5m]',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableReceivedPackets',
+          },
+          properties: [
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Received packets [5m]',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableSentBytes',
+          },
+          properties: [
+            {
+              id: 'unit',
+              value: 'decbytes',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Network throughput Send [5m]',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableReceivedBytes',
+          },
+          properties: [
+            {
+              id: 'unit',
+              value: 'decbytes',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
+            {
+              id: 'displayName',
+              value: 'Network throughput Received [5m]',
+            },
```

Comment:
Can you make this title shorter. It's hard to read now because the column is short. What about just **Network received [5m]**?